core/src/test/java/com/scalar/db/transaction/consensuscommit/TwoPhaseConsensusCommitManagerTest.java

```diff
@@ -80,8 +78,27 @@ public void setUp() throws Exception {
             coordinator,
             parallelExecutor,
             recovery,
-            commit,
-            null);
+            commit);
+  }
+
+  @Test
+  public void constructor_GroupCommitEnabledGiven_ShouldThrowException() {
+    // Arrange
+    when(config.isCoordinatorGroupCommitEnabled()).thenReturn(true);
+
+    // Act Assert
+    assertThrows(
```

Comment:
We usually use `org.assertj.core.api.Assertions.assertThatThrownBy` for this kind of test case.

Reply:
Thanks! Used it in https://github.com/scalar-labs/scalardb/pull/1797/commits/90b0ecf396e1bccb2553d11d12e59bb5236edaa4.